src/util/escapeArguments.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Escape given arguments to be compatible with Windows.
+ *
+ * @param {string[]} args
+ */
+function escapeArguments(args) {
```

Comment:
I'd move Windows check inside this function, otherwise it's not clear why only need to escape arguments on Windows and the name is too generic.